docs/install/install-eventing-with-yaml.md

```diff
@@ -131,7 +132,7 @@ Follow the procedure for the broker of your choice:
         kubectl apply -f {{ artifact(org="knative-sandbox",repo="eventing-kafka-broker",file="eventing-kafka-broker.yaml")}}
         ```
 
-    For more information, see the [Kafka broker](./../eventing/broker/kafka-broker.md) documentation.
+    For more information, see the [Kafka broker](./../eventing/broker/kafka-broker/README.md) documentation.
```

Comment:
```suggestion
    For more information, see the [Kafka broker](./../eventing/broker/kafka-broker/) documentation.
```
Pretty sure you can just leave off the default landing page .md file in these kinds of links but idk if this works the same for mkdocs?